lib/core/networking/protocol/project_build.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:json_annotation/json_annotation.dart';
+
+part 'project_build.g.dart';
+
+/// Request to build the initialized project.
+const projectBuildRequest = 'build_project';
```

Comment:
I don't see a major problem because we are using constants and not string literals at different points in our code.
An empty class would be a huge overhead imho. At the end of the day an empty class would carry the exact same information but unlike a constant it would not be meant to be used in such cases.

Reply:
The biggest problem for me is that all other messages have other casing. But you may resolve this.